src/linker/Linker/Driver.cs

```diff
@@ -568,10 +570,16 @@ public bool Run (ILogger customLogger = null)
 				// ClearInitLocalsStep
 				// OutputStep
 				//
-
-				foreach (string custom_step in custom_steps) {
-					if (!AddCustomStep (p, custom_step))
+				
+				if (custom_steps.Any()) {
+					List<Assembly> custom_step_assemblies;
+					if (!ResolveCustomStepAssemblies (custom_steps, context.Resolver.GetSearchDirectories (), out custom_step_assemblies))
 						return false;
+
+					while (custom_steps.Any()) {
+						if (!AddCustomStep (p, custom_steps.Pop (), custom_step_assemblies))
```

Comment:
I think this will change the order in which the steps are added - which may change the effective order of steps - which is an unnecessary breaking change.

Reply:
You're right, thanks for spotting this. I'll change it back.